packages/components/badge/src/Badge.tsx

```diff
@@ -0,0 +1,20 @@
+import { forwardRef, PropsWithChildren } from 'react'
+
+import { BadgeItem, BadgeItemProps } from './BadgeItem'
+
+export type BadgeProps = PropsWithChildren<Omit<BadgeItemProps, 'type'>>
+
+export const Badge = forwardRef<HTMLElement, BadgeProps>(({ children, className, ...props }) => {
+  const isStandalone = !children
+
+  return isStandalone ? (
+    <BadgeItem type="standalone" {...props} />
+  ) : (
+    <div className="relative flex">
+      {children}
+      <BadgeItem {...props} />
+    </div>
```

Comment:
- What If I want to add margin to the wrapper for example ?
- `className` not being used

Reply:
Second point is fixed. First one... I'm not sure, it could be handled in different ways... but we can add a `wrapperClassName` prop just in case.

Reply:
I think that would be better to provide a compound component API
```jsx
<BadgeContainer className="">
  <Avatar />
  <Badge className="" />
</BadgeContainer>
```

Reply:
Removed `wrapperClassName` prop, it's not mandatory for this iteration, moreover it's difficult to achieve it using a compound component pattern